Build failing due to missing six dependency

```
ImportError while importing test module '/workspace/tests/test_generate.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/local/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
tests/test_generate.py:25: in <module>
    from google.cloud import storage
/h/.local/lib/python3.9/site-packages/google/cloud/storage/__init__.py:35: in <module>
    from google.cloud.storage.batch import Batch
/h/.local/lib/python3.9/site-packages/google/cloud/storage/batch.py:27: in <module>
    import six
E   ModuleNotFoundError: No module named 'six'
```

Let's add an explicit dependency to fix the build for now.

--- a/setup.py
+++ b/setup.py
@@ -15,7 +15,7 @@
 import setuptools
 
 
-dependencies = ["click", "google-cloud-storage<2.0.0dev", "gcp-docuploader", "semver"]
+dependencies = ["click", "google-cloud-storage<2.0.0dev", "gcp-docuploader", "semver", "six"]
 
 packages = setuptools.find_packages()
 scripts = ["docuploader=docuploader.__main__:main"]

